Sming/Makefile

```diff
@@ -159,16 +156,21 @@ USER_LIBDIR = compiler/lib
 
 LIBSMING = libsming
 ifeq ($(ENABLE_SSL),1)
-	LIBSMING= libsmingssl
+	LIBSMING := $(LIBSMING)ssl
+	BUILD_BASE := $(BUILD_BASE)ssl
 endif
 
+ifeq ($(ENABLE_GDB),1)
+	LIBSMING := $(LIBSMING)d
+	BUILD_BASE := $(BUILD_BASE)d
+endif
 
 # name for the target project
 TARGET		= app
 
 CUSTOM_TARGETS ?=
 
-ARDUINO_LIBRARIES = $(shell git submodule status Libraries | cut -c2- | cut -f2 -d ' ' | sed -r 's/Libraries\/([^/]*)/Libraries\/\1\/library.properties/g' )
+ARDUINO_LIBRARIES =
```

Comment:
That change is plain wrong. Please, remove it from here. Did you use the `master` branch and tried to merge changes from it to the `develop` branch?

Reply:
Hmmm... I took a copy of the modified files from the messy fork then recreated a clean fork and copied the files back. I should have done a more granular compare of the changed files which would have shown this up. The change must have happened in the mess that was the last fork! I will sort this out in next commit.